apps/updatenotification/lib/Controller/AdminController.php

```diff
@@ -36,8 +37,8 @@ public function __construct(
 		parent::__construct($appName, $request);
 	}
 
-	private function isUpdaterEnabled() {
-		return !$this->config->getSystemValue('upgrade.disable-web', false);
+	private function isUpdaterEnabled(): bool {
+		return !$this->config->getSystemValueBool('upgrade.disable-web', false);
```

Comment:
```suggestion
		return !$this->config->getSystemValueBool('upgrade.disable-web');
```
We have that in a lot of places but I find it disturbing, default is false for boolean without needing to specify it.